solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreaker.java

```diff
@@ -66,14 +66,19 @@ protected boolean isEnabled() {
     private final int memCBThreshold;
     private final boolean cpuCBEnabled;
     private final int cpuCBThreshold;
+    private final boolean loadAverageCBEnabled;
+    private final double loadAverageCBThreshold;
 
     public CircuitBreakerConfig(final boolean enabled, final boolean memCBEnabled, final int memCBThreshold,
-                                  final boolean cpuCBEnabled, final int cpuCBThreshold) {
+                                final boolean cpuCBEnabled, final int cpuCBThreshold,
```

Comment:
Rename to highlight the metric (like you did for loadAverageCB renaming)

Reply:
I don't understand this comment.

Reply:
Please rename the circuit breaker and it's relevant parameters to reflect the metric that the circuit breaker is using

Reply:
After this change there will be tree CBs available: `memCBEnabled`, `loadAverageCBEnabled` (previously cpuCBEnabled), and `cpuCBEnabled` (the new one). So this should be correct, not?